remote_config/src/common.h

```diff
@@ -63,6 +66,25 @@ class FutureData {
   static FutureData* s_future_data_;
 };
 
+template <typename TConfigUpdate, typename TRemoteConfigError>
```

Comment:
Can the types ConfigUpdate and RemoteConfigError be used directly instead of creating a template? It looks to me like the types do not vary.

Reply:
I got an error when I tried to build the project. The error I got were along the lines of candidate function not viable: no known conversion from 'long' to 'firebase::remote_config::ConfigUpdate' for 1st argument

Reply:
Replied in the other comment about the same error.